Sming/compiler/ld/eagle.app.v6.cpp.include.ld

```diff
@@ -0,0 +1,217 @@
+/*  
```

Comment:
Well, the tests seem to fail because they do not see the new file _eagle.app.v6.cpp.include.ld_, how can I fix this? Should I split the PR (1st PR: add _eagle.app.v6.cpp.include.ld_,, 2nd PR change _eagle.app.v6.cpp.ld_)?